src/wbase/Task.h

```diff
@@ -204,17 +198,17 @@ class Task : public util::CommandForThreadPool {
     /// This functional also attempts to inform the scheduler for this
     /// `Task` that is has been cancelled. The scheduler currently does
     /// nothing in this case.
-    void cancel();
+    void cancel(bool logIt = true);
 
     /// Check if this task should be cancelled and call cancel() as needed.
     /// @return true if this task was or needed to be cancelled.
     bool checkCancelled();
 
     TaskState state() const { return _state; }
     std::string getQueryString() const;
-    bool setTaskQueryRunner(
-            TaskQueryRunner::Ptr const& taskQueryRunner);  ///< return true if already cancelled.
-    void freeTaskQueryRunner(TaskQueryRunner* tqr);
+    /// Return true if already cancelled.
```

Comment:
To stay consistent in your coding style you may do:
```
    /// @return true if already cancelled.
```

Reply:
I have found that putting @return on the first line eliminates the brief statement, which is annoying.

Reply:
Interesting. I didn't know that. Did you see this on the Doxygen-generated documentation or in your IDE (Eclipse)?